source/app/config.docker.ini

```diff
@@ -11,7 +11,7 @@ SECRET_KEY = AVerySuperSecretKey-SoNotThisOne
 SECURITY_PASSWORD_SALT = ARandomSalt-NotThisOneEither
 
 [CELERY]
-HOST = rabbitmq
+BROKER = amqp://rabbitmq
```

Comment:
**Note for ourselves**:  We probably need to add a check if the CELEREY::HOST variable is existing, otherwise we'll have issues during the upgrades. Or maybe just indicates to make the change manually in the upgrades notes.